TestFoundation/TestNSProcessInfo.swift

```diff
@@ -53,4 +54,16 @@ class TestNSProcessInfo : XCTestCase {
         processInfo.processName = originalProcessName
         XCTAssertEqual(processInfo.processName, originalProcessName, "\"\(processInfo.processName)\" not equal to \"\(originalProcessName)\"")
     }
+    
+    func test_globallyUniqueString() {
+        let uuid = NSProcessInfo.processInfo().globallyUniqueString
+        let parts = uuid._bridgeToObject().componentsSeparatedByString("-")
```

Comment:
Yep, it's to get to `componentsSeparatedByString`. What should I use instead?

Also `_bridgeToObject` is public:

```
extension String : _ObjectTypeBridgeable {
    public func _bridgeToObject() -> NSString {
        return NSString(self)
    }
...
}
```